deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -490,6 +505,8 @@ cephFileSystems:
         # in hyperconverged settings where the volume is mounted on the same node as the osds.
         csi.storage.k8s.io/fstype: ext4
 
+# TODO add documentation
```

Comment:
still in progress?

Reply:
Could we add a simple description? We don't want them left out of the docs. For example: "Settings for enabling the filesystem volume snapshot class"